### ISSUE TITLE
Oletools unable to install because of Pyparsing version problem

### ISSUE THREAD

**Issue by noasterndekel** [decalage2/oletools#762]
**Affected tool:**
general oletools

**Describe the bug**
When trying to install, pip cannot find Pyparsing in version below 3.0 (tried to install manually and upgrade pip, nothing works) and thus unable to install. 

**Console output / Screenshots**
ERROR: Could not find a version that satisfies the requirement pyparsing<3,>=2.1.0 (from oletools) (from versions: 3.0.8)
ERROR: No matching distribution found for pyparsing<3,>=2.1.0

**Version information:**
 - OS: Linux
 - OS version: Ubuntu 20.04.4 LTS
 - Python version: 3.9 
 - oletools version: latest

**Comment by jloehel** [comment]
@decalage2 Can you please explain why it was necessary to avoid pyparsing 3+ in https://github.com/decalage2/oletools/commit/750859ce1d94d782c2890c2735e2db808741b4fd. 2020 pyparsing 3+ was only available in a early alpha. Was this the reason?

**Comment by vEpiphyte** [comment]
@jloehel You can see a comment from declage2 here https://github.com/decalage2/oletools/pull/812

**Comment by jloehel** [comment]
> @jloehel You can see a comment from declage2 here #812

Thanks for the hint. I am not sure if I understand the argument here. Oletools does not depend on ViperMonkey. ViperMonkey requires Oletools. It should be enough to add the restriction to ViperMonkey. If Oletools runs with pyparsing 2.x and 3.x it's fine.

**Comment by vEpiphyte** [comment]
It is a bit confusing, since vipermonkey does have its own pyparsing constraint.

**Comment by decalage2** [comment]
Yes, it's just that I haven't touched the pyparsing code in olevba for years, and I don't know if it runs fine with the latest pyparsing versions. After some testing with different VBA samples, if it works we can relax the requirement to allow pyparsing 3.

**Comment by DManowitz** [comment]
Would also like a version that works with pyparsing 3.x.  Any word on this?
